nav2_docking/opennav_docking/src/simple_charging_dock.cpp

```diff
@@ -98,6 +103,13 @@ void SimpleChargingDock::configure(
   node_->get_parameter(name + ".staging_x_offset", staging_x_offset_);
   node_->get_parameter(name + ".staging_yaw_offset", staging_yaw_offset_);
 
+  std::string dock_direction;
+  node_->get_parameter(name + ".dock_direction", dock_direction);
+  dock_direction_ = utils::getDockDirectionFromString(dock_direction);
+  if (dock_direction_ == opennav_docking_core::DockDirection::UNKNOWN) {
+    RCLCPP_ERROR(node_->get_logger(), "Dock direction is not valid");
```

Comment:
I'd throw here, these need to be valid for this plugin

Reply:
Standard throw or new ```DockingException``` (DockPluginNotConfigured, for example?

Reply:
It actually doesn't matter, it gets caught as a `std::exception` of any form to return `false` and fail the server initialization